src/components/NavMenu/NavMenu.js

```diff
@@ -65,31 +64,6 @@ class NavMenu extends React.Component {
       Actions.setMobileMenuButtonValue('');
     }
   }
-  /**
-   * Generates the DOM for the Sticky Items that will
-   * display when the Header is in sticky mode.
-   * Adds the appropriate class based off the sticky value.
-   * @returns {Object} React DOM.
-   */
-  renderStickyNavItems() {
-    const stickyClass = HeaderStore.getIsStickyValue() ? ' active' : '';
-    return (
-      <div className={`${this.props.className}-stickyItems${stickyClass}`}>
-        <span className="lineSeparator" style={styles.lineSeparator}></span>
-        <StickyMyNyplButton
```

Comment:
This component `StickyMyNyplButton.js` should be OK to be removed as well

Reply:
Yup, thanks.